.github/workflows/integration.yml

```diff
@@ -237,8 +239,9 @@ jobs:
       - name: Set up Conda environment
         uses: conda-incubator/setup-miniconda@v3
         with:
-          activate-environment: idaes-env
-          python-version: ${{ env.DEFAULT_PYTHON_VERSION }}
+          activate-environment: ${{ env.IDAES_CONDA_ENV_NAME_DEV }}
+          python-version: ${{ matrix.python-version }}
```

Comment:
These two changes should be reversed (i.e. the only change should be adding the `miniforge-version: latest` line). 

(For reference, this is because this job's purpose is to simulate a user's environment, where IDAES is installed in non-editable mode and none of the development dependencies are available)